R/qview.R

```diff
@@ -182,33 +215,11 @@ queueView <- function(
   shiny::observe({
     index <- selected_index()
     if(is.numeric(index)) {
-      .return_values$selected_row <- .rv$queue_data %>% dplyr::slice(index)
+      .return_values$highlighted_rows <- .rv$queue_data %>% dplyr::slice(index)
```

Comment:
is this name semantically correct? is it now highlighted_row or can this be multiple somehow?

when cancelling, what is the purpose behind returning the cancelled row? should there also be a 'type' metadata attached so downstream you could know how to respond to that action. Eg a cancellation you wouldn't want it to trigger attempting to render the volcano plot etc

Reply:
To this, it is currently a single highlighted row, but it would be easy to change to multiple, I just am not sure which is better. I'll modify it to be just highlighted_row until we change it.